client/components/blazepress-widget/index.tsx

```diff
@@ -91,12 +95,20 @@ const BlazePressWidget = ( props: BlazePressPromotionProps ) => {
 					props.postId,
 					onClose,
 					source,
-					( original: string, options?: TranslateOptionsText ): string => {
-						if ( options ) {
+					(
```

Comment:
```suggestion
					translate,
```

This wrapper around `translate` seems to be redundant as it doesn't seem to do anything but pass the same arguments to the original translate function.

Reply:
Thanks, don!